closure/goog/i18n/numberformat.js

```diff
@@ -580,6 +588,140 @@ goog.i18n.NumberFormat.prototype.roundNumber_ = function(number) {
 
 
 /**
+ * This function is called when the length of the repeating digits is at least
+ * one digit. In the number grouping concept, anything left of the decimal
+ * place is followed by non-repeating digits and then repeating digits. If the
+ * pattern is #,##,###, then we first (from the left of the decimal place) have
+ * a non-repeating digit of size 3 followed by repeating digits of size 2
+ * separated by a thousand separator. If the length of the digits are six or
+ * more, there be repeating digits required. For example, the value of 12345678
+ * would be formatted as 1,23,45,678 where the repeating digit is length 2.
+ *
+ * @param {Array<string>} parts The original parts variable that built in
```

Comment:
@joeltine : Thanks for your recent comment. I'll work on it today. Was held up by daily stand up and other tasks. I'll see what I can do about the `parts` variable. Let me know if there's anything else since my involvement with the pull request exceeded my original estimated time for this.